gui/wxpython/vdigit/wxdisplay.py

```diff
@@ -856,7 +856,7 @@ def GetSelectedVertex(self, pos):
 
         if not self._validLine(line):
             return -1
-        ftype = Vect_read_line(self.poMapInfo, self.poPoints, self.poCats, line)
+        Vect_read_line(poMapInfo, self.poPoints, self.poCats, lineNearest)
```

Comment:
Where is lineNearest defined?

Reply:
This was a typo, I thought parameters would be same in both function calls. I have updated this.